.github/workflows/publish-book.yml

```diff
@@ -0,0 +1,25 @@
+name: Publish
+on:
```

Comment:
Sorry but I find it hilarious that the first file to review on this PR is a `yaml` file :laughing: